packages/backend/src/controllers/users/dto/user-response.dto.ts

```diff
@@ -1,6 +1,9 @@
 import { ApiProperty } from '@nestjs/swagger';
 
 export default class UserResponseDto {
+  @ApiProperty()
+  name: string;
+
   @ApiProperty({ required: false })
   house: string;
```

Comment:
```suggestion
  house?: string;
```
this one too, not sure why it wasn't already